jean_zay/raw/understanding-unets/unet_training.slurm

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#SBATCH --job-name=learnlets_unet     # nom du job
+#SBATCH --ntasks=1                   # nombre de tâche MPI
+#SBATCH --ntasks-per-node=1          # nombre de tâche MPI par noeud
+#SBATCH --gres=gpu:2                 # nombre de GPU à réserver par nœud
+#SBATCH --cpus-per-task=10           # nombre de coeurs à réserver par tâche
```

Comment:
Watch out here, you book 2 gpus but only 10 cpus: you need to always book `n_gpus x 10` cpus on gpu_p1

Reply:
I thought that parameter had to do with the relationship between No. GPUs and No. CPUs. So I was actually thinking about n_gpus x 10 = n_cpus. I've checked documentation and it's not that, but what you suggested.

So now it's been changed